script/ghci-flags

```diff
@@ -3,63 +3,78 @@
 
 set -e
 
-cd $(dirname "$0")/..
+cd "$(dirname "$0")/.."
 
-root="$(pwd)"
 ghc_version="$(ghc --numeric-version)"
 
-if [ "$1" == "--builddir" ]; then
-  repl_builddir="$2"
-  shift 2
-else
-  repl_builddir=dist-newstyle
-fi
-
-echo "-O0"
-echo "-ignore-dot-ghci"
-
-echo "-outputdir $repl_builddir/build/x86_64-osx/ghc-$ghc_version/semantic-0.8.0.0/build"
-echo "-odir $repl_builddir/build/x86_64-osx/ghc-$ghc_version/semantic-0.8.0.0/build"
-echo "-hidir $repl_builddir/build/x86_64-osx/ghc-$ghc_version/semantic-0.8.0.0/build"
-echo "-stubdir $repl_builddir/build/x86_64-osx/ghc-$ghc_version/semantic-0.8.0.0/build"
-
-echo "-i$root/$repl_builddir/build/x86_64-osx/ghc-$ghc_version/semantic-0.8.0.0/build/autogen"
-
-echo "-I$root/$repl_builddir/build/x86_64-osx/ghc-$ghc_version/semantic-0.8.0.0/build/autogen"
-
-echo "-i$root/semantic-analysis/src"
-echo "-i$root/semantic-ast/src"
-echo "-i$root/semantic-core/src"
-echo "-i$root/semantic-java/src"
-echo "-i$root/semantic-json/src"
-echo "-i$root/semantic-python/src"
-echo "-i$root/semantic-tags/src"
-echo "-i$root/app"
-echo "-i$root/src"
-echo "-i$root/bench"
-echo "-i$root/test"
```

Comment:
We no longer need to use absolute paths for these settings as whatever VS Code or `ghcide` bug it was which required this has now been fixed.